orderbook/Cargo.toml

```diff
@@ -24,6 +24,7 @@ primitive-types = "0.7"
 serde = { version = "1.0", features = ["derive"] }
 serde_json = "1.0"
 shared-arguments = { path = "../shared-arguments" }
+sqlx = { version = "0.4", default-features = false, features = ["runtime-tokio-native-tls", "postgres"] }
```

Comment:
I decided on sqlx because compared to tokio-postgres it has connection pooling and automatic caching of queries built in. Since this is only used in the database module it wouldn't be too hard to switch later anyway.